app/styles/app/modules/buttons.scss

```diff
@@ -326,11 +331,11 @@
 
 .jobs-item {
   .repo-main-tools {
-    transform: translate(-5px, -2px);
+    // transform: translate(-5px, -2px);
 
-    button {
-      background: transparent;
-    }
+    // button {
+    //   background: transparent;
+    // }
```

Comment:
Please remove the commented out code in case if it's not needed anymore

Reply:
Done.